recipes/xdesign/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "xdesign" %}
+{% set version = "v0.5.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/tomography/{{ name }}/archive/{{ version }}.tar.gz
```

and only put it here:
```suggestion
  url: https://github.com/tomography/{{ name }}/archive/v{{ version }}.tar.gz
```